packages/lesswrong/components/comments/CommentsSortBySelector.tsx

```diff
@@ -0,0 +1,47 @@
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import React from 'react';
+import { useLocation } from '@/lib/routeUtil';
+import qs from 'qs'
+import { isEmpty } from 'underscore';
```

Comment:
prefer using `lodash` to underscore - you'll want `import isEmpty from 'lodash/isEmpty';`